core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -244,6 +244,7 @@ object SparkEnv extends Logging {
       conf.set("spark.driver.port", rpcEnv.address.port.toString)
     } else if (rpcEnv.address != null) {
       conf.set("spark.executor.port", rpcEnv.address.port.toString)
+      logInfo(s"Starting executor with port: ${rpcEnv.address.port.toString}")
```

Comment:
Since the executor isn't actually started here, can we instead print "setting spark.executor.port to {}"